gensim/test/test_fasttext.py

```diff
@@ -325,65 +325,122 @@ def test_doesnt_match(self):
         for word in oov_words:
             self.assertFalse(word in self.test_model.wv.vocab)
         try:
-            self.test_model.doesnt_match(oov_words)
+            self.test_model.wv.doesnt_match(oov_words)
         except Exception:
             self.fail('model.doesnt_match raises exception for oov words')
 
-    def compare_with_wrapper(self, model_gensim, model_wrapper):
-        # make sure we get >=2 overlapping words for top-10 similar words suggested for `night`
-        sims_gensim = model_gensim.most_similar('night', topn=10)
-        sims_gensim_words = (list(map(lambda x: x[0], sims_gensim)))  # get similar words
-
-        sims_wrapper = model_wrapper.most_similar('night', topn=10)
-        sims_wrapper_words = (list(map(lambda x: x[0], sims_wrapper)))  # get similar words
-
-        overlap_count = len(set(sims_gensim_words).intersection(sims_wrapper_words))
+    def test_cbow_hs_training(self):
 
-        # overlap increases as we increase `iter` value, min overlap set to 2 to avoid unit-tests taking too long
-        # this limit can be increased when using Cython code
-        self.assertGreaterEqual(overlap_count, 2)
+        model_gensim = FT_gensim(size=50, sg=0, cbow_mean=1, alpha=0.05, window=5, hs=1, negative=0,
+            min_count=5, iter=5, batch_words=1000, word_ngrams=1, sample=1e-3, min_n=3, max_n=6,
+            sorted_vocab=1, workers=1, min_alpha=0.0)
 
-    def test_cbow_hs_against_wrapper(self):
```

Comment:
All `*_against_*` tests removed (because wrapper dropped), am I right?

Reply:
Correct and I have replaced them with similar test cases without the wrapper.